Docs/Configuration.tex

```diff
@@ -2986,9 +3006,9 @@ \subsection{Boot Properties}\label{miscbootprops}
   \texttt{HideAuxiliary}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Hides auxiliary entries from picker menu by default.
+  \textbf{Description}: Hides auxiliary entries from the picker menu unless overriden.
 
-  An entry is considered auxiliary when at least one of the following applies:
+  An entry is considered to be an auxiliary entry when one or more of the following apply:
```

Comment:
The original statement seems tidier in my opinion.

Reply:
Agree

Reply:
As you wish!